src/shared/components/dateRangePicker/NewDatePicker.tsx

```diff
@@ -77,8 +77,9 @@ const DatePickerMenu: FC<Props> = ({onCollapse, timeRange, timeRangeLabel}) => {
     collapse()
   }
 
+  const durationRegExp = /([0-9]+)(y|mo|w|d|h|ms|s|m|us|µs|ns)$/g
```

Comment:
Possibly, but I think 2 instances of this is fine. Also, regexes can be altered from calling `.exec`. So, if you make a constant be a regex, you have to be sure that some other code does not inadvertently call `.exec` on your constant and alter it. This will lead to very subtle and hard to find bugs.